src/main/java/type/DukeCommandTypeEnum.java

```diff
@@ -0,0 +1,22 @@
+package type;
+
+public enum DukeCommandTypeEnum {
```

Comment:
Do add a JavaDoc here as well

Reply:
Alright I've addressed all your comments, thank you!